gitlab/provider_test.go

```diff
@@ -12,9 +12,14 @@ var testAccProviders map[string]terraform.ResourceProvider
 var testAccProvider *schema.Provider
 
 func init() {
-	testAccProvider = Provider().(*schema.Provider)
-	testAccProviders = map[string]terraform.ResourceProvider{
-		"gitlab": testAccProvider,
+	if os.Getenv("TF_ACC") != "" {
```

Comment:
Did you see my last comment about adding the import and using the const? It would be good to refer to the same const that the Terraform test engine uses.

Reply:
Thanks for the suggestion-- I tried importing the resource package but for some reason it's not using the variable correctly: 
```
gitlab/provider_test.go:4:2: imported and not used: "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
gitlab/provider_test.go:15:15: undefined: TestEnvVar
```

Reply:
you need to use `resource.TestEnvVar` 👍

Reply:
This builds for me. Is this the same thing you tried? https://github.com/sfang97/terraform-provider-gitlab/pull/1

Reply:
Ah, I'd left out the `resource.` part, thanks y'all!